### PR TITLE
jdbc hw

### PR DESCRIPTION
**Pull request by MaVaYuran**

### CODE REVIEW

**Comment by MaVaYuran** [comment]
DELETE FROM users INNER JOIN ticket ON users.id=ticket.user_id; 
delete without cascade

File: src/main/java/jdbc/dao/TicketDAOImpl.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            WHERE user_id=?
+            """;
+    private static final String UPDATE_TYPE_SQL = """
+            UPDATE ticket SET ticket_type=?
+            WHERE id=?
+            """;
+
+
+    public Ticket save(Ticket t) {
+        try (var connection = ConnectionManager.open();
```
Comment (Alexmansar) [inline src/main/java/jdbc/dao/TicketDAOImpl.java]: please, use object type, not var

File: src/main/java/jdbc/dao/TicketDAOImpl.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                }
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+
+        }
+        return null;
+    }
+
+    private static Ticket buildTicket(ResultSet result) throws SQLException {
```
Comment (Alexmansar) [inline src/main/java/jdbc/dao/TicketDAOImpl.java]: move after public methods, and why its static?

File: src/main/java/jdbc/pojo/Users.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
+package jdbc.pojo;
+
+import java.sql.Timestamp;
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class Users  {
```
Comment (Alexmansar) [inline src/main/java/jdbc/pojo/Users.java:6]: why users? not user?

File: src/main/java/jdbc/pojo/Users.java [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    int id;
```
Comment (Alexmansar) [inline src/main/java/jdbc/pojo/Users.java]: what with access modifier?

File: src/main/resources/application.properties [hunk 5]
```diff
@@ -0,0 +1,3 @@
+db.url=jdbc:postgresql://localhost:5432/my_ticket_service_db
+db.username=postgres
+db.password=root
```
Comment (Alexmansar) [inline src/main/resources/application.properties:3]: 👍

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@MaVaYuran Please, fix xomments and resolve conflicts
And its correct? 
image
not into main?

File: src/main/java/jdbc/pojo/Users.java (hunk 3 above)
Comment (MaVaYuran) [reply src/main/java/jdbc/pojo/Users.java:6]: I didn't find annotation for table and decided it only in hibernate/ That's why i had to call it same as table name

File: src/main/java/jdbc/dao/TicketDAOImpl.java (hunk 2 above)
Comment (MaVaYuran) [reply src/main/java/jdbc/dao/TicketDAOImpl.java]: I extracted this with idea/ it made method static

Comment (Alexmansar) [reply src/main/java/jdbc/dao/TicketDAOImpl.java]: just check it as well

File: src/main/java/jdbc/pojo/Users.java (hunk 3 above)
Comment (Alexmansar) [reply src/main/java/jdbc/pojo/Users.java:6]: try to rename just class name to User.

**Pull request merged**